docusaurus/docs/vitest/troubleshooting.md

```diff
@@ -0,0 +1,42 @@
+---
+pagination_next: null
+pagination_prev: null
+---
+
+# Vitest Troubleshooting
+
+## JavaScript heap out of memory
+
+You can increase the memory available to Node with [`--max_old_space_size`](https://nodejs.org/api/cli.html#--max-old-space-sizesize-in-megabytes):
+
+```bash
+export NODE_OPTIONS=--max_old_space_size=4096
+
+npx knapsack-pro-jest
+
+npx knapsack-pro-cypress
+
+npx knapsack-pro-vitest
```

Comment:
suggestion

I think it would make more sense to keep `npx knapsack-pro-vitest` and remove:

```
npx knapsack-pro-jest

npx knapsack-pro-cypress
```

A similar change could be applied to Jest and Cypress troubleshooting pages.